### PR TITLE
Typography tweaks

### LINKED ISSUE #61: Presentation of inline monospace font
Inline monospace does not stand out enough. Investigate different colour/font weight combo (mkdocs and other systems use red).

### PR DESCRIPTION
**Pull request by 36degrees**
| Old | New |
| --- | --- |
| screen shot 2017-03-06 at 12 20 34 | screen shot 2017-03-06 at 12 20 47 |
| screen shot 2017-03-06 at 12 21 04 | screen shot 2017-03-06 at 12 21 07 |
| screen shot 2017-03-06 at 12 22 12 | screen shot 2017-03-06 at 12 22 13 |
| screen shot 2017-03-06 at 12 21 15 | screen shot 2017-03-06 at 12 21 19 |
| screen shot 2017-03-06 at 12 21 44 | screen shot 2017-03-06 at 12 21 46 |

Fixes #61

### CODE REVIEW

File: template/source/stylesheets/_core.scss [hunk 1]
```diff
@@ -33,6 +33,7 @@ $desktop-breakpoint: 992px !default;
 html, body {
   margin: 0;
   -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
```
Comment (fofr) [inline template/source/stylesheets/_core.scss]: 🍬

File: template/source/stylesheets/modules/_technical-documentation.scss [hunk 2]
```diff
@@ -1,19 +1,99 @@
+@mixin heading-offset($tabletTopMargin) {
```
Comment (fofr) [inline template/source/stylesheets/modules/_technical-documentation.scss]: A quick summary of what this mixin does would be useful at the start.

File: template/source/stylesheets/modules/_technical-documentation.scss [hunk 3]
```diff
@@ -1,19 +1,99 @@
...
+  // Scale margins to scale with font size on mobile (16/19ths)
```
Comment (fofr) [inline template/source/stylesheets/modules/_technical-documentation.scss]: (typo: repetition of 'scale')

File: template/source/stylesheets/modules/_technical-documentation.scss [hunk 4]
```diff
@@ -1,19 +1,99 @@
...
     // We add 2 pixels to the gutter here to ensure there's sufficient spacing between
     // an anchor link (see _anchored-heading.scss) on a heading and the left-hand pane.
     margin: 0 ($gutter + 2px) $gutter;
   }
 
-  h1, h2, h3, h4, h5, h6 {
-    padding-top: $gutter-half;
+  // Headings
+  h1 {
+    @include bold-48();
```
Comment (fofr) [inline template/source/stylesheets/modules/_technical-documentation.scss]: The empty parentheses can be omitted

**Comment by 36degrees** [comment]
Thanks for the review @fofr! Can you take another look and approve if the latest commit has addressed your comments?

**Pull request merged**